LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Gonzalo Ibars Ingman
```

Comment:
So, you've made all this entire project alone...

Reply:
Yep. 

Just kidding... Github asked for my name, I did not know I had to put everyone's name in the project, just the owner of the repository.